vm/Makefile

```diff
@@ -33,7 +33,7 @@ all: libubpf.a test
 
 ubpf_jit_x86_64.o: ubpf_jit_x86_64.c ubpf_jit_x86_64.h
 
-libubpf.a: ubpf_vm.o ubpf_jit_x86_64.o ubpf_loader.o
+libubpf.a: ubpf_vm.o ubpf_jit_x86_64.o ubpf_loader.o ubpf_run.o
```

Comment:
The "Fixes" tag should not be in this commit

Reply:
@iomartin which commit *should* it be in?

Reply:
I'd probably put it on "Add bare-metal instruction counter". This commit is just refactoring code